### DISCUSSION TITLE
Implement dfx new _project_name_ --type=azle

### DISCUSSION THREAD

**Discussion by lastmjs** [dfinity/sdk#2639] [Feature Requests]
It would be great to allow users to generate a default Azle project using `dfx new _project_name_ --type=azle`.

**Comment by viviveevee** [comment]
Hi Joardan, generally we'd like to support this, but we have a lot of open questions. @dfx-json would like to talk to you about this and has tried to reach you on Discord. Would you be willing to talk to him?

**Comment by lastmjs** [comment]
Yes that would be great, I don't see a message on Discord by I'm @lastmjs basically everywhere

**Comment by viviveevee** [comment]
IIRC you two ended up connecting. If not: please ping me again and I'll figure it out.

**Reply by lastmjs** [reply]
We did connect, seems this will be part of the plugin system
